system/Helpers/html_helper.php

```diff
@@ -306,7 +308,17 @@ function link_tag($href = '', string $rel = 'stylesheet', string $type = 'text/c
 			$link .= 'href="' . $href . '" ';
 		}
 
-		$link .= 'rel="' . $rel . '" type="' . $type . '" ';
+		if( $hreflang !== '' )
+		{
+		    $link .= 'hreflang="' . $hreflang .'" ';
+		}
+
+		$link .= 'rel="' . $rel . '" ';
+
+		if( !in_array($rel, ['alternate','canonical']) )
```

Comment:
`if (! in_array($rel, ['alternate', 'canonical']))`

Reply:
Ok, i fixed this